.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.2, 7.3, 7.4]
+                php: [7.3, 7.4]
```

Comment:
On lines 34-45 you could simplify this action by using https://github.com/ramsey/composer-install (check this as an example: https://github.com/infinum/eightshift-coding-standards/blob/main/.github/workflows/lint.yml)

Reply:
Could you open an issue for this? I'd rather not mess with this as well now.